lib/src/main/java/io/ably/lib/realtime/AblyRealtime.java

```diff
@@ -50,6 +51,7 @@ public AblyRealtime(ClientOptions options) throws AblyException {
 		super(options);
 		channels = new Channels();
 		connection = new Connection(this);
+		http.setHost(options.realtimeHost);
```

Comment:
I solved that issue here https://github.com/ably/ably-java/pull/80/files#diff-060685a947a7032665de7ac9d33bb6ffR588